RELEASING.md

```diff
@@ -137,7 +137,9 @@ $ git push origin
 ## Wait for Travis deployment...
 
 ## Release to npm
-$ npm publish
+$ lerna bootstrap # make sure dependencies are linked
+$ npm run prepublish # Build all packages
+$ lerna publish --repo-version v$SKYGEAR_VERSION # Publish all packages
```

Comment:
Since `lerna publish` will do git tag and prefix the version with `v`. So this line should not belong to here.

Also we will want to keep the unify git command on signing the tag. Let me provide a new PR.